docs/src/environments.md

```diff
@@ -11,7 +11,7 @@ In order to create a new project, create a directory for it and then activate th
 ```julia-repl
 julia> mkdir("MyProject")
 
-julia> cd("MyProject")
+julia> cd("MyProject") # we can now use "." instead of a longer relative or full path
 /Users/kristoffer/MyProject
 
 (v1.0) pkg> activate .
```

Comment:
This works with paste mode
```suggestion
julia> cd("MyProject")
/Users/kristoffer/MyProject

# we can now use "." instead of a longer relative or full path:
(v1.0) pkg> activate .
```